crates/stackable-operator/src/crd/listener/core/mod.rs

```diff
@@ -45,6 +45,8 @@ pub mod versioned {
 
         /// Preserves the client source IP and avoid a second hop for LoadBalancer and NodePort type
         /// Services, but makes clients responsible for spreading the load.
+        ///
+        /// Does not work everywhere, e.g. not on IONOS!
```

Comment:
suggestion: Don't point fingers.

```suggestion
        ///
        /// Does not work everywhere out-of-the-box. Some cloud vendors do not implement the
        /// required features and bare metal installations need proper configuration.
```